oauth2client/client.py

```diff
@@ -771,11 +771,11 @@ def _do_refresh_request(self, http):
         headers = self._generate_refresh_request_headers()
 
         logger.info('Refreshing access_token')
-        resp, content = transport.request(
+        response = transport._request(
```

Comment:
Maybe I misremembered? In this library there have been discussion about not using non-public members of a module outside of that module.